### PR TITLE
[WIP] Add scripts readme

### PR DESCRIPTION
**Pull request by canyon289**
Adding small README for scripts . Feel free to edit if you think my descriptions are terrible

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Scripts
+Various scripts that help with ArviZ maintenance
+
+#### container.ps1
```
Comment (aloctavodia) [inline scripts/README.md:4]: I think this is the same as container.sh but for windows

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+???
+
+#### container.sh
+Script that helps with building docker image, and running jupyter lab,
+jupyter notebooks, building docs, or getting a shell or testing
+
+#### create_testenv.sh
+Script that helps install arviz on whatever computer is being used
```
Comment (aloctavodia) [inline scripts/README.md:12]: ArviiZ

Comment (canyon289) [reply scripts/README.md:12]: 🤦

**Comment by canyon289** [comment]
Fixed per comments. Merging

**Comment by ahartikainen** [comment]
What did you fix?

**Pull request merged**